app/assets/javascripts/users.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
[Q]
why we need ` app/assets/images/kitten.jpg` ?

Reply:
[A] I will delete that. It was a part of the tutorial but unnecessary now.